app/viewmodels/group_member.rb

```diff
@@ -58,12 +60,8 @@ def user
   end
 
   def membership_timer
-    if @membership.end_date
-      [@membership.start_date, @membership.end_date].join(' - ')
-    else
-      "#{@membership.start_date} -"
-    end
-  end
-
+    return "#{@membership.start_date} - #{@membership.end_date}" if @membership.end_date
 
+    "#{@membership.start_date} -"
```

Comment:
Or maybe: `"#{@membership.start_date} - #{@membership.end_date''}"` cause if `end_date` is nil then it will be an empty string.